.openpublishing.redirection.json

```diff
@@ -1341,6 +1341,122 @@
             "source_path": "docs/framework/add-ins/walkthrough-create-extensible-app.md",
             "redirect_url": "/previous-versions/dotnet/netframework-4.0/bb788290(v%3dvs.100)"
         },
+        {
+            "source_path": "docs/framework/app-domains/assemblies-and-side-by-side-execution.md",
+            "redirect_url": "/dotnet/standard/assembly/assemblies-and-side-by-side-execution"
+        },
+        {
+            "source_path": "docs/framework/app-domains/assemblies-in-the-common-language-runtime.md",
+            "redirect_url": "/dotnet/standard/assembly/assemblies-in-the-common-language-runtime"
+        },
+        {
+            "source_path": "docs/framework/app-domains/assembly-contents.md",
+            "redirect_url": "/dotnet/standard/assembly/assembly-contents"
```

Comment:
I think we should run SEO tests on this content. I think that these new paths will introduce SEO problems because they have duplicate words in the path. These could probably be simplified to /dotnet/standard/assembly/contents

@rpetrusha what do you think?